moveit_planners/ompl/ompl_interface/src/model_based_planning_context.cpp

```diff
@@ -47,11 +48,21 @@
 #include <moveit/profiler/profiler.h>
 #include <moveit/utils/lexical_casts.h>
 
+#include <ompl/config.h>
 #include <ompl/base/samplers/UniformValidStateSampler.h>
 #include <ompl/base/goals/GoalLazySamples.h>
 #include <ompl/tools/config/SelfConfig.h>
 #include <ompl/base/spaces/SE3StateSpace.h>
 #include <ompl/datastructures/PDF.h>
+// \todo remove when ROS Melodic and older are no longer supported
```

Comment:
```suggestion
// TODO remove when ROS Melodic and older are no longer supported
```

Small style standard that wasnt documented. Now it is:
https://github.com/ros-planning/moveit.ros.org/pull/379/files